lib/engine/game/g_18_eus/step/bidbox_auction.rb

```diff
@@ -5,128 +5,130 @@
 module Engine
   module Game
     module G18EUS
-      module BidboxAuction
-        include Engine::Step::Auctioner
+      module Step
+        module BidboxAuction
+          include Engine::Step::Auctioner
+
+          def actions(entity)
+            actions = []
+            actions << 'bid' << 'pass'
+            actions.concat(super).uniq
+          end
+
+          def setup
+            # This sets the initial value of @bids
+            setup_auction
+            super
+
+            @bid_actions = 0
+            @bids = @round.bids if @round.bids
```

Comment:
@round.bids should be setup in this module. See `def round_state` for examples throughout the code base.

Reply:
@bids is setup in setup_auction (auctioner.rb) which is called in setup.  

@round.bids has to stay here because setup is called after round_state.  Removed setting @bids based on @round.bids.  Added def round_state to define @round.bids.